ComponentKit/Utilities/CKSectionedArrayController.mm

```diff
@@ -130,6 +132,8 @@ - (void)enumerateObjectsInSectionAtIndex:(NSInteger)sectionIndex usingBlock:(CKS
 
 - (CKArrayControllerOutputChangeset)applyChangeset:(CKArrayControllerInputChangeset)changeset
 {
+  verifyChangeset(changeset, _sections);
```

Comment:
We might, but we can do that when we get there. Right now, the crashes we're seeing are in the sectioned array controller so we should start there.

We'd also need a different algorithm I think since transactional changesets are different objects.